scripts/apps/authoring/authoring/directives/SendItem.tsx

```diff
@@ -920,6 +924,12 @@ export function SendItem($q,
                     stage = _.find(scope.stages, {_id: scope.selectedDesk.incoming_stage});
                 }
 
+                if (appConfig.ui && appConfig.ui.defaultStage != null) {
+                    stage = scope.stages
```

Comment:
This would overwrite previously set values. Can you restructure the code so it wouldn't do it?